src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -704,6 +797,23 @@ TEST_F(ResumeCtrlTest, IsApplicationSaved) {
   EXPECT_TRUE(res_ctrl->IsApplicationSaved(policy_app_id, device_id));
 }
 
+TEST_F(ResumeCtrlTest, OnAppRegistrationStart) {
+  const std::string kPolicyAppId = "policy_app_id";
+  const std::string kDeviceId = "device_id";
```

Comment:
@LitvinenkoIra, Please note, that not global const variables should be called as regular ones.